src/tests/TestCrypt/TestCrypt.cpp

```diff
@@ -251,6 +251,22 @@ void TestCrypt::xexstarAttack() {
 	// Make sure we detected the attack
 	QVERIFY(failed_encrypt);
 	QVERIFY(failed_decrypt);
+
+	// The assumption that critical packets do not turn up by pure chance turned out to be incorrect
+	// since digital silence appears to produce them in mass.
+	// So instead we now modify the packet in a way which should not affect the audio but will
+	// prevent the attack.
+	QVERIFY(cs.ocb_encrypt(src, encrypted, 2 * AES_BLOCK_SIZE, nonce, enctag));
+	QVERIFY(cs.ocb_decrypt(encrypted, decrypted, 2 * AES_BLOCK_SIZE, nonce, dectag));
+
+	// Tags should match
+	for (int i = 0; i < AES_BLOCK_SIZE; ++i) {
+		QCOMPARE(enctag[i], dectag[i]);
+	}
+
+	// Actual content should have been change such that the critical block is no longer all 0.
```

Comment:
```suggestion
	// Actual content should have been changed such that the critical block is no longer all 0.
```

Reply:
@Johni0702 what speaks against applying this change? :thinking:

Reply:
Every time I amend the commit, GitHub adds another mention to every issue and PR referenced from the commit. So I'm waiting till all feedback is in as to not unnecessarily spam those issues.

Reply:
Roger :+1: